packages/components/src/Components/TagCount/TagCount.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './tagCount.scss';
+
+const TagCount = ({count, onTagClick, ...props}) => {
```

Comment:
You should include `className` in here, so consumers can apply their own classes.